docs/how_tos/layered_cookiecutter.rst

```diff
@@ -0,0 +1,39 @@
+====================
+Layered Cookiecutter
+====================
+
+To learn more about why we are using layered cookiecutters, see the decision 0003-layered-cookiecutter.rst.
+
+Cookiecutters using this approach
+---------------------------------
+- cookiecutter-django-app
+- cookiecutter-django-ida
+- cookiecutter-python-library
+
+How it works
+------------
+
+Cookiecutter allows you to define {pre, post}_gen_project.py files that run at beginning of the folder creation and at end of folder creation. 
+
+For the layered approach, we use the post_gen_project.py file to add the bottom layers to the output folder. The script creates each layer in folder(s) called: placeholder_repo_name_#(num based on how many templates are created) and then moves all resulting files to the correct location. 
+
+Emphasis: the layers are placed in high to low order, so the topmost(the most specific) layer is placed first. Each subsequent layer adds files without replace, so if the file already exists, it is not replaced by file from bottom layer. 
+
+For example: for cookiecutter-django-app(CDA), the CDA specific files/folders are created first, then files from django-template, and then finally files from python-template.
```

Comment:
1. `cookiecutter-django-app(CDA)` -> `cookiecutter-django-app (CDA)`
2. Does this mean that the CDA can't remove a file that is provided by python-template?  It can't say it doesn't want something?

Reply:
CDA can delete unnecessary files created by python-template has created its files.  CDI(cookiecutter-django-ida) does this.